app/org/sagebionetworks/bridge/models/surveys/SurveyRule.java

```diff
@@ -133,12 +173,29 @@ public Builder withEndSurvey(Boolean endSurvey) {
             }
             return this;
         }
+        public Builder withDisplayIf(Boolean displayIf) {
+            if (Boolean.TRUE.equals(displayIf)) {
+                this.displayIf = displayIf;
+            }
```

Comment:
Why do we need to check that this is true? Are we concerned that people might accidentally call withDisplayIf(true), and then later call withDisplayIf(false)? If so, why is leaving it to true more intuitive than leaving it to false?

Alternatively, if the goal is to default to false, it would be better for the lower case b boolean (since we'd only have two states anyway) in both the class and the builder (explicitly initialize it to false for documentation/clarity). This way, it'll be consistent with all the other builders, be easier to reason about, and easier to test.

If we _do_ keep this logic, we'll want to add unit tests to make sure that, once set, we can't clear it with withDisplayIf(false) or withDisplayIf(null).

(Here and in withDisplayUnless().)